src/sentry/buffer/base.py

```diff
@@ -44,6 +57,29 @@ def get_hash(
     def get_sorted_set(self, key: str, min: float, max: float) -> list[tuple[int, datetime]]:
         return []
 
+    def push_to_sorted_set(self, key: str, value: list[int] | int) -> None:
+        return None
+
+    def push_to_hash(
```

Comment:
Btw I'm not sure if we use `RedisBuffer` in self hosted, it's worth checking. In dev we use this, so we should make sure that alerts and so on continue working when we're using `Buffer`. Maybe that just means keeping "organizations:process-slow-alerts" around permanently, and have it disabled in dev by default, but enabled elsewhere?

Alternatively, we could write this to just process in realtime when we enqueue.